dvjsharma/Drawn2Shoe
# docs(users): Modified documentation

## Description

This PR aims to unify and finalise documentation for Drawn2Shoe

### Changes

* Added `LICENSE`
* Added more components into `README.md`
* Modified `CONTRIBUTORS.md` for better clarity and rule sets
* Removed `LEARN.md` since it's content is already covered by others 


### How to Test

* Changes are visible in the PR, check for any errors
* Pull the branch and preview the changes

CC @ShashankShekhar07
